src/main/webapp/app/entities/flair-bi/filter/filter-element-grpc.component.js

```diff
@@ -113,7 +115,7 @@
             };
             filterParametersService.saveSelectedFilter(filterParameters);
             vm.isActive(filter);
-            if(isFavouriteFilter())
+            if (isFavouriteFilter())
```

Comment:
there is no change in code..if u want to format the code...pls create separate PR